content/docs/esc/environments/dynamic-environment-variables.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Dynamic environment variables
+title_tag: Dynamic environment variables | Pulumi ESC
+h1: Dynamic environment variables
+meta_desc: Pulumi ESC allows you to securely run commands with managed environment variables using the esc run command, without exporting them to your shell.
+menu:
+  pulumiesc:
+    identifier: esc-dynamic-environmeant-variables
+    parent: esc-environments
+    weight: 4
+---
+
+The Pulumi ESC CLI includes a [`run` command](/docs/esc-cli/commands/esc_run/) that allows you to run commands with Pulumi ESC managed environment variables, without exporting them to your shell.
+
+## Running commands with environment variables
```

Comment:
I don't think we need this heading here and suggest we use it for the H1 instead.

Reply:
added to H1, removed h2